### PR TITLE
Make Gift Aid tick box more obvious

### LINKED ISSUE #279: Make the Gift Aid tick box more obvious
To make the text in bold and at the bottom of billing details rather than at top when making a donation

### PR DESCRIPTION
**Pull request by pro-Nate**
fix #279

### CODE REVIEW

File: js/src/donations/PaymentForm.js [hunk 1]
```diff
@@ -158,11 +158,21 @@ export class PaymentForm_ extends React.Component {
...
-                     onChange={gift_aid => this.setState({gift_aid})}/>
 
-              <Collapse isOpen={this.state.gift_aid}>
+              <StripeForm submitting={this.state.submitting}
+                          payment_state={this.state.payment}
+                          setPaymentState={this.setPaymentState}/>
+
+              <hr/>
+
+              <strong>
```
Comment (samuelcolvin) [inline js/src/donations/PaymentForm.js:168]: do you need `strong` here as well as `b` above?

**Review by samuelcolvin — COMMENTED** [review]
otherwise looks good.

File: js/src/donations/PaymentForm.js (hunk 1 above)
Comment (pro-Nate) [reply js/src/donations/PaymentForm.js:168]: Bootstrap adds an element with a 'text-muted' class around the label text. Another css class would be needed to override and this was cleaner.

**Review by samuelcolvin — APPROVED** [review]
LGTM

**Pull request merged**